packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -4207,9 +4207,9 @@ export type ResourceFetchResult = {
 
 // @beta
 export interface RichTextEditBoxOptions extends RichTextEditorOptions {
-    messagesInlineImages?: Record<string, AttachmentMetadataInProgress[]>;
-    onCancelInlineImageUpload?: (imageId: string, messageId: string) => void;
-    onInsertInlineImage?: (imageUrl: string, imageFileName: string, messageId: string) => void;
+    messagesInlineImagesWithProgress?: Record<string, AttachmentMetadataInProgress[]>;
+    onInsertInlineImage?: (imageAttributes: Record<string, string>, messageId: string, imageFileName?: string) => void;
```

Comment:
Should we create an object here so Contoso will handle everything correctly (messageId and imageFileName have the same type)? 

I'm also wondering if we need imageFileName as a separate attribute here if imageAttributes should already include it

Reply:
> I'm also wondering if we need imageFileName as a separate attribute here if imageAttributes should already include it

Had the same thought and I think @palatter's concern was that it's not as discoverable when embedded in the imageAttributes under a custom data-name